GomokuClient/packages/gomoku-core/src/context/SignlaRProvider.tsx

```diff
@@ -12,35 +12,27 @@ import { JsonHubProtocol } from "@microsoft/signalr";
 import { typedStorage } from "@/shared/lib/utils";
 import { useAuthToken } from "@/context/AuthContext";
 import { useAuth } from "@clerk/clerk-react";
-import { CellValue } from "@/hooks/useBoard";
+import {
+  SignalClientMessages,
+  SignalHubInterfaces,
+  SignalRClientService,
+  SignalServerMessages,
+} from "@/api";
 
 interface PlayerJoinedGameServerMessage {
   userName: string;
 }
 
-interface GameStartedServerMessage {
-  isMyMoveFirst: boolean;
-}
-
-interface PlayerMadeMoveServerMessage {
-  playerId: string;
-  tile: TileItem;
-  placedTileColor: CellValue;
-}
-
-interface SendMessageServerMessage {
-  gameId: string;
-  user: string;
-  message: string;
-}
-
-interface TileItem {
-  x: number;
-  y: number;
-}
-
-interface GameHubError {
-  message: string;
+interface SignalREventHandlers {
+  onPlayerJoined?: (message: PlayerJoinedGameServerMessage) => void;
```

Comment:
Was not able to find that event, do we have it?

```ts
interface PlayerJoinedGameServerMessage {
  userName: string;
}
```

Reply:
Will add in upcoming PR